### PR TITLE
Portworx module support

### PR DESCRIPTION
**Pull request by umarali-nagoor**
Screenshot 2021-10-13 at 6 53 02 PM
Screenshot 2021-10-14 at 6 09 40 PM

### CODE REVIEW

**Comment by umarali-nagoor** [comment]
Test Results:
Screenshot 2021-11-29 at 2 34 37 PM
Screenshot 2021-11-29 at 2 32 22 PM
39118/143906941-f2ac7b58-208a-4bf9-bad8-48311dd26584.png">

**Comment by umarali-nagoor** [comment]
Screenshot 2021-11-26 at 4 04 36 PM
>

**Comment by umarali-nagoor** [comment]
Screenshot 2021-11-26 at 4 04 36 PM

File: variables.tf [hunk 1]
```diff
@@ -1,24 +1,79 @@
-/********************************************************************
+##############################################################################
+# Account Variables
+##############################################################################
 
-This file is used to declare ROOT varaibles.
+variable "install_storage" {
+  type        = bool
```
Comment (hkantare) [inline variables.tf:6]: Why we need this variable

File: variables.tf [hunk 2]
```diff
@@ -1,24 +1,79 @@
...
+  type        = string
+}
+
+variable unique_id {
+  description = "Unique identifiers for all created resources"
+  type        = string
+}
+
+variable cluster {
+  description = "Name of existing roks cluster"
```
Comment (hkantare) [inline variables.tf:22]: change the description to remove roks

File: variables.tf [hunk 3]
```diff
@@ -1,24 +1,79 @@
...
+  type        = string
+}
+
+variable "kube_config_path" {
+  description = "Path to the k8s config file: ex `~/.kube/config`"
```
Comment (hkantare) [inline variables.tf:27]: ~/.kube/config remove this and say /tmp

File: variables.tf [hunk 4]
```diff
@@ -1,24 +1,79 @@
...
+##############################################################################
+
+variable capacity {
+  description = "Capacity for all block storage volumes provisioned in gigabytes"
+  type        = number
+  default     = 100
+}
+
+variable profile {
+  description = "The profile to use for this volume."
```
Comment (hkantare) [inline variables.tf:49]: change the description of profile and capacity according to terraform reosures docs

File: volume.tf [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+resource ibm_is_volume volume {
+
+  depends_on = [
+    data.ibm_is_subnet.subnet
+  ]
+
+  count = var.install_storage ? length(data.ibm_container_vpc_cluster.cluster.workers) : 0
+
+  name           = "vol-workerss-${element(split("-", data.ibm_container_vpc_cluster.cluster.workers[count.index]), 4)}"
+  profile        = "10iops-tier"
```
Comment (hkantare) [inline volume.tf:34]: why this profile hardcoded?

**Pull request merged**